src/test/java/com/faforever/client/game/GamesTableControllerTest.java

```diff
@@ -58,7 +58,7 @@ public void test() throws Exception {
 
   @Test
   public void testKeepsSorting() {
-    preferencesService.getPreferences().setGameListSorting(FXCollections.observableArrayList(
+    preferencesService.getPreferences().getGameListSorting().setAll(FXCollections.observableArrayList(
```

Comment:
Since there's a varargs signature of `setAll` you don't need to wrap it into `FXCollections.observableArrayList` :)